bigquery_etl/util/common.py

```diff
@@ -50,9 +51,8 @@ def random_str(length: int = 12) -> str:
 
 def render(sql_filename, format=True, template_folder="glean_usage", **kwargs) -> str:
     """Render a given template query using Jinja."""
-    env = Environment(
-        loader=PackageLoader("bigquery_etl", f"{template_folder}/templates")
-    )
+    file_loader = FileSystemLoader(f"{template_folder}/templates")
+    env = Environment(loader=file_loader)
```

Comment:
Moving this into `sql_generators` means it's not part of the `bigquery_etl` package anymore, so use `FileSystemLoader` instead.